packages/cloud_security_posture/changelog.yml

```
@@ -1,4 +1,15 @@
 # newer versions go on top
+- version: "0.0.21"
+  changes:
+    - description: Adjust findings data-stream mappings to fit ECS conventions
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/3775
+    - description: Turned off dynamic mappings of findings data-stream
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/3775
+    - description: Added default pipeline to findings data-stream
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/3775
```

I'm not sure listing the same PR multiple times is encouraged.
```suggestion
    - description: |
            Adjust findings data-stream mappings to fit ECS conventions
            Turned off dynamic mappings of findings data-stream
            Added default pipeline to findings data-stream
      type: enhancement
      link: https://github.com/elastic/integrations/pull/3775
```

---

saw several packages doing the same in their changelog, thought to give it a try

---

tbh, my previous try with version `0.0.19` combining multiple prs to the same changelog was not a successful one. Because of the post-merge process happens when the version is being create for the first time, it is not being updated afterwards